gennav/envs/scan_env.py

```diff
@@ -1,17 +1,16 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
could you remove this? No need to make this executable

Reply:
Yeah sure

Reply:
Yeah sure